mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -78,7 +78,9 @@ Packages=
         gcc
         # Required to run systemd-networkd-tests.py
         python3
+        python3-selinux
         iproute2
         dnsmasq-base
         wireguard-tools
         isc-dhcp-server
+        netlabel-tools
```

Comment:
Ah, you added these packages for systemd-networkd-tests.py. Sorry, please forget my previous comment, and also add them for other distro?

Reply:
Added. Arch doesn't seem to have netlabel_tools packaged.